tests/infrastructure/tekton/conftest.py

```diff
@@ -262,21 +308,93 @@ def pipelinerun_from_pipeline_template(
 
 
 @pytest.fixture()
-def final_status_pipelinerun(pipelinerun_from_pipeline_template):
-    try:
-        for sample in TimeoutSampler(
-            wait_timeout=TIMEOUT_50MIN,
-            sleep=TIMEOUT_1MIN,
-            func=lambda: pipelinerun_from_pipeline_template.instance.status.conditions[0],
-        ):
-            if sample and sample["status"] != Resource.Condition.Status.UNKNOWN:
-                # There are 3 conditions.status possible : Unknown, False, True.
-                LOGGER.info(f"PipelineRun Condition : {sample}")
-                return sample
-
-    except TimeoutExpiredError:
-        LOGGER.error(
-            f"Pipelinerun: {pipelinerun_from_pipeline_template.name} , "
-            f"Preparing for VM teardown due to Timeout Error.Last available sample: {sample}"
-        )
-        raise
+def quay_disk_uploader_secret(custom_pipeline_namespace):
+    with Secret(
+        name="quay-disk-uploader-secret",
+        namespace=custom_pipeline_namespace.name,
+        accesskeyid=base64_encode_str(os.environ["QUAY_ACCESS_KEY_TEKTON_TASKS"]),
+        secretkey=base64_encode_str(os.environ["QUAY_SECRET_KEY_TEKTON_TASKS"]),
+    ) as quay_disk_uploader_secret:
+        yield quay_disk_uploader_secret
+
+
+@pytest.fixture()
+def created_fedora_dv_for_disk_uploader(admin_client, golden_images_namespace, custom_pipeline_namespace):
+    with create_dv(
+        dv_name="fedora-dv-disk-uploader",
+        namespace=custom_pipeline_namespace.name,
+        source=PVC,
+        source_pvc=DataSource(
+            name=OS_FLAVOR_FEDORA, namespace=golden_images_namespace.name
+        ).instance.spec.source.pvc.name,
+        source_namespace=golden_images_namespace.name,
+        size="35Gi",
```

Comment:
do you need 35Gi or default size can be used? If this is needed, then please add a comment explaining why.

Reply:
@dbasunag ^